src/test/regress/expected/multi_multiuser.out

```diff
@@ -536,7 +536,7 @@ ERROR:  could not receive file "base/pgsql_job_cache/job_0042/task_000001/p_0000
 -- different user should not be able to fetch partition file
 SET ROLE usage_access;
 SELECT worker_fetch_partition_file(42, 1, 1, 1, 'localhost', :worker_1_port);
-WARNING:  could not open file "base/pgsql_job_cache/job_0042/task_000001/p_00001.18007": No such file or directory
+WARNING:  could not open file "base/pgsql_job_cache/job_0042/task_000001/p_00001.17981": No such file or directory
```

Comment:
Why did the user OID change here?

Reply:
Probably from being run on pg12. [It gets normalized](https://github.com/citusdata/citus/blob/169d2f193fb8e58f5425ac51668e93404501fc52/src/test/regress/bin/normalize.sed#L72)